packages/core/src/ahoi_items/NavigationItem.ts

```diff
@@ -0,0 +1,30 @@
+import { ANCHORCLASS, ARROWCLASS, NAVIGATIONCLASS } from "../constants";
+import { EOnboardingStages } from "../interfaces";
+import { AAhoiItem } from "./AAhoiItem";
+
+export default class NavigationItem extends AAhoiItem {
+    constructor(parent: HTMLElement, iconClass: string, stage: EOnboardingStages, htmlTitle: string, onClick: () => void) {
+      super(parent, NAVIGATIONCLASS, onClick, {iconClass, stage, htmlTitle})
+      this.instance.classList.add(this.getItemClass(), "hidden");
+    }
+
+    protected clickHandler() {
+      super.clickHandler();
+      document.querySelectorAll(`.${NAVIGATIONCLASS}`).forEach((item) => item.classList.add("hidden"));
+      document.querySelectorAll(`.${ANCHORCLASS}, .${ARROWCLASS}`).forEach((item) => item.classList.remove("hidden"));
```

Comment:
```suggestion
      parent.querySelectorAll(`.${NAVIGATIONCLASS}`).forEach((item) => item.classList.add("hidden"));
      parent.querySelectorAll(`.${ANCHORCLASS}, .${ARROWCLASS}`).forEach((item) => item.classList.remove("hidden"));
```

`document.querySelectorAll()` will cause side effects if VisAhoi is used multiple times on the same page.

Maybe `parent` works, but it would be better that a separate `Navigation` class manages the state and show/hide of children.